actors/test/commit_post_test.go

```diff
@@ -331,3 +333,307 @@ func TestCommitPoStFlow(t *testing.T) {
 		assert.True(t, networkStats.TotalPledgeCollateral.GreaterThan(big.Zero()))
 	})
 }
+
+func preCommitSectors(t *testing.T, v *vm.VM, batchSize int, worker, mAddr address.Address, sealProof abi.RegisteredSealProof) ([]abi.SectorNumber, []*miner.SectorPreCommitOnChainInfo) {
+	sectorNumberBase := 100
+	precommits := make([]*miner.SectorPreCommitOnChainInfo, 0)
```

Comment:
If it's going to be empty, use `precommits := []*miner.SectorPreCommitOnChainInfo{}`